CMakeLists.txt

```diff
@@ -173,7 +176,7 @@ set (VC_LIB_HDR ${VC_LIB_HDR1} ${VC_LIB_HDR2} ${VC_LIB_HDR3} )
 include_directories( ${CMAKE_CURRENT_SOURCE_DIR}/include )
 
 add_library( ${VC_LIB_NAME} STATIC ${VC_LIB_SRC} ${VC_LIB_HDR} )
-target_link_libraries( ${VC_LIB_NAME} ${VC_LIB_DEPS} )
+target_link_libraries( ${VC_LIB_NAME} ${VC_LIB_DEPS} z pthread tinfo)
```

Comment:
can we make a self_explanatory variable for these new additions?

Reply:
Probably they were not required. I'd remove them if you agree with me.